calico_node/startup/startup.go

```diff
@@ -497,23 +509,22 @@ func configureIPPools(client *client.Client) {
 	// Ensure there are pools created for each IP version.
 	if !ipv4Present {
 		log.Debug("Create default IPv4 IP pool")
-		createIPPool(client, ipv4Cidr, ipv4IpipModeEnvVar)
+		createIPPool(client, ipv4Cidr, ipv4IpipModeEnvVar, evaluateENVBool("CALICO_IPPOOL_NAT_OUTGOING", "NAT_Outgoing disabled for Default IPv4 Pool."))
```

Comment:
Please change the environment variable names to be IP version specific.
ie. CALICO_IPV4POOL_NAT_OUTGOING